localisation/wc_religions.csv

```diff
@@ -252,8 +252,12 @@ CAPITALIST;Capitalist;Capitaliste;;;;;;;;;;;;x;
 pandaren_religion_group;Chi;chi;;;;;;;;;;;;x;
 lightbound;Lightbound;;;;;;;;;;;;;x;
 lightbound_DESC;A heretical offshoot of the naaru teachings, the adherents to the Lightbound doctrine are dedicated to spread the influence of the Light wherever they possibly can and convert the unbelievers by any means necessary, even forcibly.;;;;;;;;;;;;;x;
+throsic;Throsic;;;;;;;;;;;;;x;
+throsic_DESC;Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare that they later named Thros, the Blighted Lands, where they cultivated a potent form of death magic with bastardized druidic elements as a result of numerous attempts to bewitch the souls of the fallen soldiers into stone constructs to use against the human settlers. As a result, the Throsic form of necromancy prefers binding the souls of the dead into durable and unrelenting stone revenants or wicker golems, while the traditional form of corpse reanimation is much more rare. Its influence is usually spread by the native covens into wildlife and the living. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through an ritualistic act of slitting their own throat in their ultimate devotion to their horrific masters from beyond the veil.;;;;;;;;;;;;;x;
```

Comment:
```suggestion
throsic_DESC;Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare, that they later named Thros, the Blighted Lands, where they cultivated a potent form of death magic with bastardized druidic elements. This form of magic resulted from numerous attempts to bewitch the souls of fallen soldiers into stone constructs in order to use against them against human settlers. Following this, the Throsic form of necromancy prefers binding the souls of the dead into durable and unrelenting stone revenants or wicker golems, while the traditional form of corpse reanimation is much rarer. Its influence is usually perpetuated by the native covens into wildlife and the living. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through a ritualistic act of slitting their own throat in ultimate devotion to their horrific masters from beyond the veil.;;;;;;;;;;;;;x;
```

Reply:
Haven't checked ingame yet, but I think, this text might be "a little" too long for a religion description...